### PR TITLE
Addition of ClickNCollect resources for export of orders

### PR DESCRIPTION
**Pull request by flug**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -156,6 +156,59 @@ Note: to test the plugin locally, see [CONTRIBUTING.md](CONTRIBUTING.md)
...
+            - "@property_accessor"
+            - "@doctrine.orm.entity_manager"
+            - "@sylius.service.address_concatenation"
+            - "@sylius.exporter.orm.hydrator.orders"
+  
+
+```
+
+```yaml
+
```
Comment (dunglas) [inline README.md]:
```suggestion
# config/services.yaml
services:
```

File: README.md [hunk 2]
```diff
@@ -156,6 +156,59 @@ Note: to test the plugin locally, see [CONTRIBUTING.md](CONTRIBUTING.md)
 11. Configure the pick up locations, the available time slots, and how many people you can safely serve in parallel
 12. Create a dedicated shipping method
 13. Optionally, configure an online payment method from the admin (Stripe and PayPal are supported out of the box)
+14. Optionally, export commands with ClickNCollect, install this [export library](https://packagist.org/packages/friendsofsylius/sylius-import-export-plugin) and add this : 
+
+```yaml
+
```
Comment (dunglas) [inline README.md]:
```suggestion
# config/services.yaml
services:
```

File: README.md [hunk 3]
```diff
@@ -156,6 +156,59 @@ Note: to test the plugin locally, see [CONTRIBUTING.md](CONTRIBUTING.md)
...
+    sylius.exporter.pluginpool.orders:
+        class: FriendsOfSylius\SyliusImportExportPlugin\Exporter\Plugin\PluginPool
+        arguments:
+            - ["@sylius.exporter.plugin.resource.orders"]
+            - ["Number", "ClickNCollect_Location_Name", "ClickNCollect_Location_Countrycode", "ClickNCollect_Location_Street", "ClickNCollect_Location_City",  "ClickNCollect_Location_PostCode", "ClickNCollect_Pin", "ClickNCollect_CollectionTime"  ]
+        
+
```
Comment (dunglas) [inline README.md]: Remove empty lines

File: README.md [hunk 4]
```diff
@@ -156,6 +156,59 @@ Note: to test the plugin locally, see [CONTRIBUTING.md](CONTRIBUTING.md)
 11. Configure the pick up locations, the available time slots, and how many people you can safely serve in parallel
 12. Create a dedicated shipping method
 13. Optionally, configure an online payment method from the admin (Stripe and PayPal are supported out of the box)
+14. Optionally, export commands with ClickNCollect, install this [export library](https://packagist.org/packages/friendsofsylius/sylius-import-export-plugin) and add this :
```
Comment (dunglas) [inline README.md]: I would move this in another section.

File: src/Exporter/Plugin/OrderResourcePlugin.php [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+declare(strict_types=1);
+
+namespace CoopTilleuls\SyliusClickNCollectPlugin\Exporter\Plugin;
+
+use CoopTilleuls\SyliusClickNCollectPlugin\Entity\ClickNCollectShipmentInterface;
+use FriendsOfSylius\SyliusImportExportPlugin\Exporter\Plugin\OrderResourcePlugin as BaseOrderResourcePlugin;
+use Sylius\Component\Core\Model\OrderInterface;
+use Sylius\Component\Core\Model\Shipment;
+
+class OrderResourcePlugin extends BaseOrderResourcePlugin
```
Comment (dunglas) [inline src/Exporter/Plugin/OrderResourcePlugin.php]:
```suggestion
final class OrderResourcePlugin extends BaseOrderResourcePlugin
```

**Review by dunglas — CHANGES_REQUESTED** [review]

**Review by toofff — APPROVED** [review]

**Comment by toofff** [comment]
Thank's @flug

**Pull request merged**